src/styles/_catalog.scss

```diff
@@ -1,22 +1,32 @@
+$card-width: 200px;
+
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  grid-template-columns: repeat(4, $card-width);
+  gap: 48px;
 
   max-width: 944px;
   margin: 0 auto;
 
+  @media (max-width: 1023px) {
+    grid-template-columns: repeat(3, $card-width);
+  }
+
+  @media (max-width: 767px) {
+    grid-template-columns: repeat(2, $card-width);
+  }
+
+  @media (min-width: 0) and (max-width: 487px) {
+    grid-template-columns: repeat(1, $card-width);
+  }
+
   &__card {
-    display: flex;
+    display: grid;
     justify-content: center;
     align-items: center;
-    width: 200px;
+    width: $card-width;
```

Comment:
You have already set the width of the cell in the grid container itself. You can get rid of it in the element itself